src/test/java/seedu/findvisor/model/person/PhoneTest.java

```diff
@@ -27,23 +27,20 @@ public void isValidPhone() {
         // invalid phone numbers
         assertFalse(Phone.isValidPhone("")); // empty string
         assertFalse(Phone.isValidPhone(" ")); // spaces only
-        assertFalse(Phone.isValidPhone("91")); // less than 3 numbers
+        assertFalse(Phone.isValidPhone("9123456")); // less than 8 numbers
         assertFalse(Phone.isValidPhone("phone")); // non-numeric
         assertFalse(Phone.isValidPhone("9011p041")); // alphabets within digits
-        assertFalse(Phone.isValidPhone("9312 1534")); // spaces within digits
```

Comment:
Is there a reason to remove this test case? I think the test case should remain to assert it's condition on whitespaces?

Reply:
The trimming of phone number is not done by phone itself so a space in phone number will still assertFalse. That said, since it allows spaces, this test case has been "shifted" to ParserUtilTest as ParserUtil is the class that trims the phone number of spaces.

Reply:
I think the test case should still remain to indicate that spaces within digits are not valid phone numbers? Then `ParserUtils` can have its own test case that allows for spaces and trims the whitespace

Reply:
I agree with @KohGuanZeh, perhaps it would be better to keep the test case as a phone number is valid only if has digits and complies to the validation regex. 

I think a good way to think about it is to isolate the `Phone` class away from its dependencies such as input parsing, as such whitespaces will be already removed in this case and any leftover whitespaces or other non-numeric input would be deemed as invalid for our `Phone` object.